terraform/cloud-platform/main.tf

```diff
@@ -127,18 +127,13 @@ resource "auth0_client" "components" {
   app_type    = "regular_web"
 
   callbacks = [
-    "https://prometheus.apps.${local.cluster_base_domain_name}/oauth2/callback",
-    "${terraform.workspace == local.live_workspace ? format("https://prometheus.%s/oauth2/callback", local.live_domain) : ""}",
-    "https://alertmanager.apps.${local.cluster_base_domain_name}/oauth2/callback",
-    "${terraform.workspace == local.live_workspace ? format("https://alertmanager.%s/oauth2/callback", local.live_domain) : ""}",
+    "${terraform.workspace == local.live_workspace ? format("https://prometheus.%s/oauth2/callback", local.live_domain) : "https://prometheus.apps.${local.cluster_base_domain_name}/oauth2/callback"}",
+    "${terraform.workspace == local.live_workspace ? format("https://alertmanager.%s/oauth2/callback", local.live_domain) : "https://alertmanager.apps.${local.cluster_base_domain_name}/oauth2/callback"}",
     "https://prometheus.apps.${local.cluster_base_domain_name}/redirect_uri",
```

Comment:
Is this and the next one used anywhere? (the `/redirect_uri` path URLs)

Reply:
honestly don't remember

Reply:
removed